package.json

```diff
@@ -24,7 +25,7 @@
     "@primer/css": "^14.4.0",
     "@snapshot-labs/lock": "github:snapshot-labs/lock#master",
     "@snapshot-labs/snapshot-spaces": "github:snapshot-labs/snapshot-spaces#master",
-    "@snapshot-labs/snapshot.js": "github:snapshot-labs/snapshot.js#master",
```

Comment:
Once this PR (https://github.com/snapshot-labs/snapshot.js/pull/235) is merged, this have to be revert.

Reply:
Yes we can merge snapshot-labs/snapshot.js#235 once we merge the branch `safesnap` to production